components/basic-transformers/src/main/java/org/datacleaner/beans/codec/XmlEncoderTransformer.java

```diff
@@ -19,24 +19,18 @@
  */
 package org.datacleaner.beans.codec;
 
-import javax.inject.Named;
-
-import org.datacleaner.api.Categorized;
-import org.datacleaner.api.Configured;
-import org.datacleaner.api.Description;
-import org.datacleaner.api.InputColumn;
-import org.datacleaner.api.InputRow;
-import org.datacleaner.api.OutputColumns;
-import org.datacleaner.api.Transformer;
+import com.google.common.escape.Escaper;
+import com.google.common.xml.XmlEscapers;
+import org.datacleaner.api.*;
 import org.datacleaner.components.categories.StringManipulationCategory;
 import org.datacleaner.data.MockInputColumn;
 
-import com.google.common.escape.Escaper;
-import com.google.common.xml.XmlEscapers;
+import javax.inject.Named;
```

Comment:
I second this remark. For me it's not an absolute blocker, but it is quite unfortunate that there are so many files in the diff where the only change is a reorganization of imports.